docs/sources/flow/reference/components/loki.source.azure_event_hubs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: loki.source.azure_event_hubs
+---
+
+# loki.source.azure_event_hubs
+
+`loki.source.azure_event_hubs` receives Azure Event Hubs messages by making use of an Apache Kafka
+endpoint on Event Hubs. For more information, see
+the [Azure Event Hubs documentation](https://learn.microsoft.com/en-us/azure/event-hubs/azure-event-hubs-kafka-overview).
+
+To learn more about streaming Azure logs to an Azure Event Hubs, you can see
+this [tutorial](https://learn.microsoft.com/en-us/azure/active-directory/reports-monitoring/tutorial-azure-monitor-stream-logs-to-event-hub).
```

Comment:
We prefer defaulting to slightly more descriptive link titles, as recommended by Google's style guide on [Link text](https://developers.google.com/style/link-text): 

```suggestion
To learn more about streaming Azure logs to an Azure Event Hubs, refer to 
Microsoft's tutorial on how to [Stream Azure Active Directory logs to an Azure event hub](https://learn.microsoft.com/en-us/azure/active-directory/reports-monitoring/tutorial-azure-monitor-stream-logs-to-event-hub).
```